bundles/org.openhab.ui/web/src/pages/settings/things/channel/channel-copy.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <f7-page @page:afterin="onPageAfterIn" name="channel-copy">
+    <f7-navbar :title="channel.label" :subtitle="thing.label" back-link="Cancel">
```

Comment:
I'd prefer to just have "Copy channel" as title.

Reply:
This hasn't been addressed yet, can you please do that?

Reply:
Do you mean that

` <f7-navbar title="Copy channel" :subtitle="thing.label" back-link="Cancel">`
instead of 
channel.label ?

Reply:
done